scalekit/events.py

```
@@ -0,0 +1,54 @@
+from typing import Optional
+
+from scalekit.core import CoreClient
+from scalekit.v1.events.events_pb2 import *
```

Ruff (0.15.21)

[error] `from scalekit.v1.events.events_pb2 import *` used; unable to detect undefined names

(F403)